packages/modules/imodel-browser/package.json

```diff
@@ -72,19 +72,19 @@
     "rollup": "^2.42.4",
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "rollup-plugin-postcss": "^4.0.0",
-    "rollup-plugin-typescript2": "^0.30.0",
+    "rollup-plugin-typescript2": "^0.36.0",
     "stylelint": "^13.6.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.18.0",
-    "ts-jest": "^26.0.0",
-    "tslib": "^2.1.0",
+    "ts-jest": "^27.1.4",
+    "tslib": "^2.6.2",
     "typescript": "^4.2.3"
   },
   "peerDependencies": {
     "@itwin/itwinui-icons-react": "^2.2.0",
-    "@itwin/itwinui-react": "^2.11.4",
+    "@itwin/itwinui-react": "^3.0.4",
```

Comment:
why is itwinui a peer dep??

Reply:
This pkg is used in iTwinStudio and studio is always close to latest versions, Putting it as a peer so this pkg can be used with studio even when studio is using 3.1 or later of itwinui. I made it peer due to the same issue we faced with 2.x

Reply:
i dont understand? why cant there be multiple instances of itwinui in studio? i believe the entire point have itiwnui@3 was so its eaiser for multiple versions to coexist

Reply:
Never tried with 3.x, this change was done for 2.x as multiple instance weren't working back then. Just out of curiosity, is it bad to have these as peer since I was under the impression pkgs should put @itwin related pkgs as peer so consumer app can take care of them?

Reply:
This applies to packages in lockstep, not every packages in @itwin...

Reply:
if you put here iui as peer and only v3, you block a bunch of apps consuming new version of this package, as they are still on 2.
If you want peer here, put interval so it actually supports v2 and v3. but if thats the case, y ou need to verify, your updates WILL WORK with v2, which looking at random css changes seem will not.
So you need to move it to deps.

Off topic-why studio is broken using v2?? I dont udnerstand that part..

Reply:
That makes sense. Thanks!

Studio was complaining about two different pkg versions of itwinui, when I put the version same as this pkg is using, studio was working fine, thats why I made it a peer. But it looks like with 3.x this problem hopefully will not occur

Reply:
Moved it back to dependencies